test/metabase/query_processor_test/timezones_test.clj

```diff
@@ -248,7 +248,7 @@
       (for [i (range 366)]
         [(u.date/add start-date :day i)])]]))
 
-(defmethod driver/database-supports? [:driver ::extract-week-of-year-us]
+(defmethod driver/database-supports? [::driver/driver ::extract-week-of-year-us]
   [_driver _feature _database]
   true)
```

Comment:
Before, this incorrectly returned false:
```
(mt/with-driver :h2 (driver/database-supports? :h2 ::extract-week-of-year-us (mt/db)))
```
Now it returns true.

Reply:
Given how easy this mistake was to make, should we not add a failing test that this fixes?

Reply:
I wouldn't, I think the mistake was made when we switched from `:driver` to `::driver/driver`. Now that the keyword is global I can't see how we could repeating that mistake.